# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fs/test/gtest/api/TestFileSystemDirectoryHandle.cpp
# Commit: e4b3f3d8ba5f
# Full Hash: e4b3f3d8ba5fd951bf79a2014dcee109255c865f
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2022-08-31 21:54:47
# Regressor Bug: 1786465
# File Overlap Count: 1
# Description:
#   Bug 1786465 - Rename POriginPrivateFileSystem to PFileSystemManager; r=dom-storage-reviewers,jesup
#   
#   The child actor will be held by FileSystemManager on the child side and the
#   parent actor already holds FileSystemDataManager, so it makes sense to rename
#   the protocol to PFileSystemManager.
# ==============================================================================

diff -r 3b0da4b4bb83 -r e4b3f3d8ba5f dom/fs/test/gtest/api/TestFileSystemDirectoryHandle.cpp
--- a/dom/fs/test/gtest/api/TestFileSystemDirectoryHandle.cpp	Wed Aug 31 09:34:48 2022 +0000
+++ b/dom/fs/test/gtest/api/TestFileSystemDirectoryHandle.cpp	Wed Aug 31 09:34:48 2022 +0000
@@ -14,7 +14,7 @@
 #include "mozilla/dom/FileSystemDirectoryIterator.h"
 #include "mozilla/dom/FileSystemHandle.h"
 #include "mozilla/dom/FileSystemHandleBinding.h"
-#include "mozilla/dom/OriginPrivateFileSystemChild.h"
+#include "mozilla/dom/FileSystemManagerChild.h"
 
 #include "mozilla/UniquePtr.h"
 #include "nsIGlobalObject.h"